### ISSUE TITLE
[BUG] undefined symbol: peerConnectionOnSenderBandwidthEstimation

### ISSUE THREAD

**Issue by dave-malone** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1161]
**Logging**

[KVS GStreamer Master] Using trickleICE by default
[2021/06/02 19:17:08:2303] N: lws_tls_client_create_vhost_context: doing cert filepath 5904f97438-certificate.pem.crt
[2021/06/02 19:17:08:2310] N: Loaded client cert 5904f97438-certificate.pem.crt
[2021/06/02 19:17:08:2311] N: lws_tls_client_create_vhost_context: doing private key filepath
[2021/06/02 19:17:08:2314] N: Loaded client cert private key 5904f97438-private.pem.key
[KVS GStreamer Master] Created signaling channel testchannel
[KVS Gstreamer Master] Finished initializing GStreamer
[KVS Gstreamer Master] Streaming video only
[KVS GStreamer Master] streaming type video-only[KVS GStreamer Master] KVS WebRTC initialization completed successfully
[KVS GStreamer Master] Signaling client created successfully
[KVS GStreamer Master] Signaling client connection to socket established
[KVS Gstreamer Master] Beginning streaming...check the stream over channel testchannel
./amazon-kinesis-video-streams-webrtc-sdk-c/build/samples/kvsWebrtcClientMasterGstSample: symbol lookup error: ./amazon-kinesis-video-streams-webrtc-sdk-c/build/samples/kvsWebrtcClientMasterGstSample: undefined symbol: peerConnectionOnSenderBandwidthEstimation

**Describe the bug**

I have the SDK and the sample applications built and running on a Raspberry Pi v4 with a Camera enabled. Using previous versions of the sample applications, I had no problem running the kvsWebrtcClientMasterGstSample application. I checked out the latest code today, and upon running the kvsWebrtcClientMasterGstSample application, making changes in samples/Samples.h, uncommenting `#define IOT_CORE_ENABLE_CREDENTIALS  1`, everything initializes just fine with my device certificates. However, once I open up the Media Playback Viewer in the AWS console for my given signaling channel name, I'm unable to view the stream, and the above errors are logged on the device.

If I comment the following lines in samples/Common.c and rebuild the sample applications, I'm able to re-initialize the sample app on my device and view the video stream in the AWS Console with no issues:  https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/4c11bbd1f3d46f11c86246c4f22490210a606271/samples/Common.c#L490-L491

**SDK version number**
Branch: Master
Git commit ID: 3e638736df68d0e3d3a7ff93cc8f97fac3d6c024

**Open source building**
n/a

**To Reproduce**
See steps listed in the description 

**Expected behavior**
Expect to be able to view my video stream in the AWS Console from my device without issues

**Screenshots**
n/a

**Desktop (please complete the following information):**
n/a

**Smartphone (please complete the following information):**
n/a

**Additional context**

**Comment by MushMal** [comment]
Seems like a stale shared library issue. Check which version of the webRTC library you are linking against and ensure its rebuilt.

**Comment by dave-malone** [comment]
Would you please provide some specific steps to try? The project repo was cloned and built as a new build in a distinct directory from previous builds.

**Comment by MushMal** [comment]
The build instructions in the README specify "build" directory under the source project root as the output directory where the build artifacts will reside. So, those are separated per cloned repository. The other questions is how do you deploy the libraries to your RPi. Are you building directly in RPi? Can you reproduce this on a commodity OS/platform?

**Comment by dave-malone** [comment]
I am building directly on the Raspberry Pi 4. As mentioned previously, this issue did not exist and the samples worked exactly as expected with prior commits. Unfortunately I just deleted the version I was building against that worked previously so I cannot retrieve the commit ID that was working.

It took me a bit, but I tested a net new build on an Ubuntu instance running Ubuntu 18.04 and it does seem to work without issue. 

However, the big difference between the builds is that on the Ubuntu instance, I'm not using the newly merged PR for the AWS IoT Credentials provider. Instead I'm using an AWS access key pair.
